server/api/region.go

```diff
@@ -557,6 +560,74 @@ func (h *regionsHandler) GetTopSize(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+// @Tags region
+// @Summary Accelerate regions schedule in given range, only receive hex format for keys
+// @Accept json
+// @Param body body object true "json params"
+// @Param limit query integer false "Limit count" default(256)
+// @Produce json
+// @Success 200 {string} string "The regions in given range have been improve schedule priority"
+// @Failure 400 {string} string "The input is invalid."
+// @Router /regions/accelerate-schedule [post]
+func (h *regionsHandler) AccelerateRegionsScheduleInRange(w http.ResponseWriter, r *http.Request) {
+	rc := getCluster(r.Context())
+	var input map[string]interface{}
+	if err := apiutil.ReadJSONRespondError(h.rd, w, r.Body, &input); err != nil {
+		return
+	}
+	parseKey := func(name string, input map[string]interface{}) (string, error) {
+		k, ok := input[name]
+		if !ok {
+			return "", fmt.Errorf("missing %s", name)
+		}
+		key, ok := k.(string)
+		if !ok {
+			return "", fmt.Errorf("bad format %s", name)
+		}
+		returned, err := hex.DecodeString(key)
+		if err != nil {
+			return "", fmt.Errorf("split key %s is not in hex format", name)
+		}
+		return string(returned), nil
+	}
+
+	startKey, err := parseKey("start_key", input)
+	if err != nil {
+		h.rd.JSON(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	endKey, err := parseKey("end_key", input)
+	if err != nil {
+		h.rd.JSON(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	limit := 256
+	if limitStr := r.URL.Query().Get("limit"); limitStr != "" {
+		var err error
+		limit, err = strconv.Atoi(limitStr)
+		if err != nil {
+			h.rd.JSON(w, http.StatusBadRequest, err.Error())
+			return
+		}
+	}
+	if limit > maxRegionLimit {
+		limit = maxRegionLimit
+	}
+
+	regions := rc.ScanRegions([]byte(startKey), []byte(endKey), limit)
+	if len(regions) > 0 {
+		regionsIDList := make([]uint64, 0, len(regions))
+		for _, region := range regions {
+			regionsIDList = append(regionsIDList, region.GetID())
+		}
+		rc.AddSuspectRegions(regionsIDList...)
+	}
+
+	h.rd.JSON(w, http.StatusOK, nil)
```

Comment:
How about using Text and add a message for it?

Reply:
updated.